docs: migration of Nomad Autoscaler docs with cluster updates. (#8315)
hashicorp/nomad: website/pages/docs/autoscaling/policy.mdx

---
layout: docs
page_title: Scaling Policies
sidebar_title: Policy
description: >
  Scaling policies describe the target resource desired state and how to
  perform calculations to ensure the current state reaches the desired.
---

# Nomad Autoscaler Scaling Policies

Nomad Autoscaler scaling policies can be configured via the
[task group scaling stanza][jobspec_scaling_stanza] or by configuration
files stored on disk.

## Top Level Options

- `enabled` - A boolean flag that allows operators to administratively disable a
  policy from active evaluation.

- `min` - The minimum running count of the targeted resource. This can be 0 or any
  positive integer.

> - `max` - The maxim running count of the targeted resource. This can be 0 or any

Reviewer: ```suggestion
- `max` - The maximum running count of the targeted resource. This can be 0 or any
```

  positive integer.

## `policy` Options

- `cooldown` - A time interval after a scaling action during which no additional
  scaling will be performed on the resource. It should be provided as a duration

>   (i.e.: "5s", "1m" etc). If omitted the configuration value

Reviewer: ```suggestion
  (e.g.: "5s", "1m"). If omitted the configuration value
```

  [policy_default_cooldown][policy_default_cooldown_agent] from the agent will
  be used.

- `evaluation_interval` - Defines how often the policy is evaluated by the

>   Autoscaler. It should be provided as a duration (i.e.: "5s", "1m" etc). If

Reviewer: ```suggestion
  Autoscaler. It should be provided as a duration (e.g.: "5s", "1m"). If
```
